metis-indexing/src/test/resources/europeana_record_rdf.xml

```diff
@@ -0,0 +1,1060 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When using example files for testing, I would say try to make them as small as possible to test exactly what you want to test. Coming back to it later to fix/update something will be possibly very difficult

Reply:
Reduced the size, by removed redundant information.